src/app/lib/layoutGrid/index.js

```diff
@@ -129,13 +130,13 @@ export const layoutGridItemMedium = css`
 
   grid-column: 1 / span 6;
   @media (max-width: ${GEL_GROUP_2_SCREEN_WIDTH_MAX}) {
-    padding: 0 ${props => (props.padding && props.padding.group2) || 'auto'};
+    padding: 0 ${props => deepGet(['padding', 'group2'], props) || ''};
```

Comment:
It'd be clearer to change to this:
```suggestion
    padding: 0 ${({padding}) => (padding && padding.group2) || ''};
```